package.json

```diff
@@ -36,6 +37,7 @@
 		"cookie": "^0.5.0",
 		"discord-oauth2": "^2.9.0",
 		"prisma-exclude": "^1.0.2",
+		"semver": "^7.3.8",
```

Comment:
Can you revert these changes and the package-lock changes and use: `npm i semver @types/semver --save --package-lock=false`